Changelog

```diff
@@ -1,17 +1,57 @@
-Version 3.3, 2020-XX-XX
+Version 3.3, *2020-XX-XX*
+
+  Features:
+    * New token type: WebAuthn/FIDO2 token (#1468)
+      * WebAuthn hardware tokens can now be used with privacyIDEA
+    * New token type: Indexed Secret. (#1986)
+      * A challenge-response token that asks the user for positions from a list of secrets.
+    * *OPEN New sophisticated logging facility (#1580)*
```

Comment:
As we talked I currently would remove this and add things that were commited after *now* add to the changelog in a 2nd round.

Reply:
ok. Removed open issues.